server.js

```diff
@@ -0,0 +1,38 @@
+const express = require('express')
+const urllib = require('urllib')
+const app = express()
+const path = require('path')
+app.use(express.static(path.join(__dirname, 'node_modules')))
+app.use(express.static(path.join(__dirname, 'dist')))
+
+const teamToIDs = {
+    "lakers": "1610612747",
+    "warriors": "1610612744",
+    "heat": "1610612748",
+    "suns": "1610612756"
+}
+
+app.get('/teams/:teamName', function(request, mainResponse) {
```

Comment:
you shoule keep api calls standardized. they should always be
`app.get('/teams/:teamName', function(req, res) {`

do not rename the parameters